Bojzen-I-Mitten/Stort-spelprojekt
# Scene bug fixes

Fixed a large bug that prevented scenes from being completely loaded.

To test:

* Add whatever you want in the scene.
* RenderComponents for days
* Prefabs, lots of prefabs.
* Save the scene
* Restart and load it. It should not bug out.
